report.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Do we need this file ?

Reply:
Yes, we don't need that. Removed it now. Thanks.